src/Main.elm

```diff
@@ -55,20 +59,23 @@ init flags url navKey =
 
 subscriptions : Model -> Sub Msg
 subscriptions model =
-    Sub.none
+    Sub.batch
+        [ Ext.Browser.Navigation.onUrlRequest OnUrlRequest
+        , Ext.Browser.Navigation.onUrlChange OnUrlChange
```

Comment:
`Browser.application` automagically sends this in to our `update`. We have to manually subscribe via port now